sonarts-core/tests/rules/noLogicalOrInSwitchCaseRule/noLogicalOrInSwitchCaseRule.lint.ts

```diff
@@ -0,0 +1,21 @@
+function bar() {}
+function baz() {}
+
+function foo(x: any) {
+  switch (x) {
+    case 1:
+      return 1;
+    case 2 || 3:
+      // ^^^^^^ {{Explicitly specify 2 separate cases that fall through; currently this case clause only works for "2".}}
+      return 2;
+    case "a" || "b" || "c" || "d":
+      // ^^^^^^^^^^^^^^^^^^^^^^^^ {{Explicitly specify 4 separate cases that fall through; currently this case clause only works for "a".}}
+      return 3;
+    case bar() || baz(): // OK, not literals
```

Comment:
@stas-vilchik Could you try to raise issue here as well? I still think it's a confusing code, and we excluded it from JS due to the noise